src/styles/_catalog.scss

```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
```

Comment:
You always need to add transition property

Reply:
@OlehMy fixed, check again, please.